Tasks/DownloadGitHubReleaseV0/package-lock.json

```diff
@@ -367,18 +393,15 @@
           }
         },
         "@types/minimatch": {
-          "version": "2.0.29",
-          "resolved": "https://registry.npmjs.org/@types/minimatch/-/minimatch-2.0.29.tgz",
+          "version": "https://registry.npmjs.org/@types/minimatch/-/minimatch-2.0.29.tgz",
```

Comment:
shouldn't version be a number here?

Reply:
@v-nmursapall  I'm not sure, that's how npm generated that 😄  Regenerating does the same thing. Used node 10.24.1 and npm  6.14.12

Reply:
Regenerated package lock. now version present as number